specs/_features/eip7594/das-core.md

```diff
@@ -75,15 +75,17 @@ The following values are (non-configurable) constants used throughout the specif
 
 | Name | Value | Description |
 | - | - | - |
-| `DATA_COLUMN_SIDECAR_SUBNET_COUNT` | `32` | The number of data column sidecar subnets used in the gossipsub protocol |
+| `DATA_COLUMN_SIDECAR_SUBNET_COUNT` | `128` | The number of data column sidecar subnets used in the gossipsub protocol |
 
 ### Custody setting
 
 | Name | Value | Description |
 | - | - | - |
-| `SAMPLES_PER_SLOT` | `8` | Number of `DataColumnSidecar` random samples a node queries per slot |
-| `CUSTODY_REQUIREMENT` | `1` | Minimum number of subnets an honest node custodies and serves samples from |
-| `TARGET_NUMBER_OF_PEERS` | `70` | Suggested minimum peer count |
+| `SAMPLES_PER_SLOT` | `16` | Number of `DataColumn` random samples a node queries per slot |
```

Comment:
There is no such thing as `DataColumn`

Reply:
There is a `DataColumnSidecar`. It's here:

https://github.com/ethereum/consensus-specs/blob/dev/specs/_features/eip7594/das-core.md#datacolumnsidecar

Reply:
Nice catch!